java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -1057,7 +1058,8 @@ private void initStringsToVTCodes() {
                 int code = 0;
                 try {
                     code = field.getInt(null);
-                } catch (IllegalAccessException | IllegalArgumentException e) {
+                }
```

Comment:
@pabender @rhwood Do we have a convention for braces in try-catch structures?  I'm used to to t.he previous `} catch ().. {` form, but if we're going to switch over consistently.

@geowar1 Is this an IDE setting that's changing these?

Reply:
@bobjacobsen I believe the old form is our standard.

Reply:
Our standards and the standard we reference are the old form. I think the NetBeans and Eclipse IDEs default to the old form, but I’ll double check that recent changes did not lead to NetBeans defaulting to this change here.

Reply:
Definitely a bug in NetBeans; The preference was off and it's still adding the newline. Toggling it on/off didn't make any difference (in the actual sources; the preference pane behaves as expected). I'll have to fix these using BBEdit (since NetBeans re-formats on save).

Reply:
Fixed (new lines removed).